src/platform/baytrail/include/platform/lib/shim.h

```diff
@@ -55,7 +55,16 @@
 #define SHIM_SSP5_DIVH		0x114
 
 #define SHIM_SHIM_BEGIN		SHIM_CSR
-#define SHIM_SHIM_END		SHIM_HDMC
+
+#if defined CONFIG_BAYTRAIL
+
+#define SHIM_SHIM_END		SHIM_SSP2_DIVH
+
+#elif defined CONFIG_CHERRYTRAIL
+
+#define SHIM_SHIM_END		SHIM_SSP5_DIVH
+
+#endif
```

Comment:
SHIM_BEGIN and SHIM_END are not used. Should they both be removed?

If we keep them, the SHIM_END cannot be used to determine a size, it should be SSPx_DIVH+8 (which would also match the definitions for the kernel).

Reply:
Hmm, is there any benefit to keeping them other than verbosity in the range of the window? Typically I am against excess definitions except when it comes to ABIs, but this is kind of a gray area.